bin/_test-helpers.sh

```diff
@@ -7,7 +7,7 @@ set +e
 ##### Test setup helpers #####
 
 export default_test_names=(deep external-issuer helm-deep helm-upgrade uninstall upgrade-edge upgrade-stable)
-export all_test_names=(cluster-domain cni-calico-deep multicluster "${default_test_names[*]}")
+export all_test_names=(cluster-domain cni-calico-deep external-prometheus-deep multicluster "${default_test_names[*]}")
```

Comment:
I think it's fine to keep `external-prometheus-deep` in `default_test_names` because it doesn't require re-creating the cluster(s) unlike `cluster-domain`, `cni-calico-deep` and `multicluster`